src/kbc_modules/uri.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! ResourceUri is the identification information of all resources that need to be
+//! obtained from `get_resource` endpoint. Also, `kid` field in an
+//! [`super::AnnotationPacket`] of `decrypt_payload` should also follow this.
+
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+const RESOURCE_ID_ERROR_INFO: &str =
+    "invalid kbs uri, should be kbs://<addr-of-kbs>/<repo>/<type>/<tag>";
```

Comment:
`invalid kbs uri` -> `invalid kbs resource uri` is better, I think.

Reply:
Thanks, fixed it !